tests/Unit/Schema/Directives/CanDirectiveTest.php

```diff
@@ -156,6 +156,41 @@ public function testProcessesTheArgsArgument(): void
         ')->assertErrorCategory(AuthorizationException::CATEGORY);
     }
 
+    public function testSendInputArgumentToPolicy(): void
+    {
+        $this->be(new User);
+        $this->schema = '
+        type Query {
+            users(key1: String, key2: String, key3: String): [User]!
+                @can(ability:"severalArgs", input: true)
+                @field(resolver: "'.$this->qualifyTestResolver('resolveUser').'")
+        }
+        
+        type User {
+            name: String
+        }
+        ';
+
+        $variables = [
+            'key1' => 'foo',
+            'key2' => 'foo2',
+            'key3' => 'foo3',
+        ];
+
+        $this->postGraphQL(
```

Comment:
Using variables is really verbose.

Please use `->graphQL()` and literal query syntax.